app/serializers/review_serializer.rb

```diff
@@ -0,0 +1,14 @@
+class ReviewSerializer < BaseSerializer
+  type :review
+
+ 	attributes(
+		:id,
```

Comment:
**[rubocop]** <sub>reported by [reviewdog](https://github.com/reviewdog/reviewdog) :dog:</sub><br>[Correctable] Layout/IndentationStyle: Tab detected in indentation.